packages/components/src/Layout/Layout.component.js

```diff
@@ -54,22 +58,41 @@ function Layout({
 		default:
 			Component = OneColumn;
 	}
+
+	let withInjectDrawers = [];
```

Comment:
That name doesn't really make sense.
Maybe that entire block of `if/else` should be extracted into a properly named function.